monitoring/uss_qualifier/visualize_configuration.py

```diff
@@ -0,0 +1,36 @@
+#!env/bin/python3
```

Comment:
Just a small suggestion, it would be nice to add a script in `bin/` to run this inside a container.

Reply:
Sounds good -- done.